plugins/quantum_k_means/schemas.py

```diff
@@ -78,9 +81,9 @@ class InputParametersSchema(FrontendFormBaseSchema):
         metadata={
             "label": "Number of clusters",
             "description": "Number of clusters that shall be found.",
-            "input_type": "text",
+            "input_type": "number",
         },
-        validate=ma.validate.Range(min=1, min_inclusive=True, error="The number of clusters must be at least 1")
+        validate=ma.validate.Range(min=1, min_inclusive=True)
```

Comment:
Why remove the error texts?

Reply:
The default message has the same amount of information, but is more generic, which fits better with the other error messages.